src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -841,47 +638,24 @@ public String captureGesture(String gestureName) {
     StringBuffer script = new StringBuffer();
     Date date = new Date();
 
-    String indentSpace = "";
     script.append("# - " + date + " - Captured gesture :\n");
 
     if (gestureName != null) {
-      indentSpace = "  ";
       script.append(String.format("def %s():\n", gestureName));
     }
 
-    if (head != null) {
-      script.append(indentSpace);
-      script.append(head.getScript(getName()));
-    }
-
-    if (leftArm != null) {
-      script.append(indentSpace);
-      script.append(leftArm.getScript(getName()));
-    }
-    if (rightArm != null) {
-      script.append(indentSpace);
-      script.append(rightArm.getScript(getName()));
-    }
-
-    if (leftHand != null) {
-      script.append(indentSpace);
-      script.append(leftHand.getScript(getName()));
-    }
-    if (rightHand != null) {
-      script.append(indentSpace);
-      script.append(rightHand.getScript(getName()));
-    }
+    try {
```

Comment:
this should be replaced with a pose from the ServoMixer...

Reply:
yeah ... i don't like the script processing in InMoov - but wasn't ready to do that change yet...